vscode/src/debugger/activate.ts

```diff
@@ -55,13 +55,17 @@ function registerCommands(context: vscode.ExtensionContext) {
         startDebugging(resource, { name: "Debug Q# File", stopOnEntry: true }),
     ),
     vscode.commands.registerCommand(
-      `${qsharpExtensionId}.debugEditorContentsWithCircuit`,
+      `${qsharpExtensionId}.runEditorContentsWithCircuit`,
       (resource: vscode.Uri) =>
-        startDebugging(resource, {
-          name: "Debug Q# File",
-          stopOnEntry: true,
-          showCircuit: true,
-        }),
+        startDebugging(
+          resource,
+          {
+            name: "Run file and show circuit diagram",
```

Comment:
Does this `name` get used for anything? Does it show up in any UI?

Reply:
I've never seen it show up. Which probably explains why I didn't even have it correct in the first place.